automation_tools/satellite6/upgrade/__init__.py

```diff
@@ -86,11 +88,17 @@ def setup_products_for_upgrade(product, os_version):
     if product == 'client' or product == 'longrun':
         logger.info('Setting up Clients ....')
         clients6, clients7 = satellite6_client_setup()
+        global_dict = {
```

Comment:
Rename 'global_dict' to 'setups_dict' else a third person wont understand, what are the further uses of it.